README.md

```diff
@@ -1 +1,78 @@
 # the-next-web-clone
+
+Responsive Web Design
+
+<h2><b>Week 2 Media query project</b></h2>
+<hr>
+<p>The assignment is the Responsive Web Design</p>
+<hr>
+<img src="./img/the-next-web-screen-shoot.JPG">
```

Comment:
- [ ] As requested by the previous reviewer, some details, i.e the image on the readme, is not showing. Please fix that.